qwik-graphql-tailwind/src/components/file-explorer/index.tsx

```diff
@@ -36,15 +35,7 @@ export const FileExplorer = component$(() => {
                 <DocumentIcon className={styles.iconFile} />
               )}
             </div>
-            <a
-              href={`${basePath}/${item.type}/${branch}/${item.path.replace(/\./g, SPECIAL_PERIOD_CHAR)}?forkCount=${
-                store.info.data?.forkCount || 0
-              }&watcherCount=${store.info.data?.watcherCount || 0}&stargazerCount=${
-                store.info.data?.stargazerCount || 0
-              }&issuesCount=${store.info.data?.openIssueCount || 0}&prCount=${
-                store.info.data?.openPullRequestCount || 0
-              }`}
-            >
+            <a href={`${basePath}/${item.type}/${branch}/${item.path}`}>
```

Comment:
Just a thought - would it maybe make more sense to use `Link` here instead of `a`?

Reply:
hmm, not sure if I tried that, to see if the issues we had with Link were fixed, but I will give it one last spin.